### PR TITLE
Remove old exchange rate

### PR DESCRIPTION
**Pull request by lomamech**
- Remove old exchange rate Service
- Remove loss record who using exchange service

@jniles

### CODE REVIEW

File: client/src/partials/exchange/modal.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ angular.module('bhima.controllers')
 
 ExchangeModalController.$inject = [
   'ExchangeRateService', 'CurrencyService',
-  'SessionService', '$uibModalInstance', 'data', 'exchange'
+  'SessionService', '$uibModalInstance', 'data'
 ];
 
-function ExchangeModalController(Rates, Currencies, Session, $uibModalInstance, data, exchange) {
+function ExchangeModalController(Rates, Currencies, Session, $uibModalInstance, data) {
```
Comment (jniles) [inline client/src/partials/exchange/modal.js:9]: Does this modal still work without the `exchange` service?  Why was it used?

**Comment by jniles** [comment]
@lomamech, je vais completer l'analyse ce soir quand je peux executer le test e2e.

File: client/src/partials/exchange/modal.js (hunk 1 above)
Comment (lomamech) [inline client/src/partials/exchange/modal.js:9]: the exchange service was called but not used, this is why I deleted

Comment (jniles) [inline client/src/partials/exchange/modal.js:9]: Excellent.

**Comment by jniles** [comment]
End to end tests pass, everything looks good to me!

**Pull request merged**